lib/stdio/validate.js

```diff
@@ -12,7 +12,7 @@ const validateObjectTransformReturn = function * (optionName, chunk) {
 const validateStringTransformReturn = function * (optionName, chunk) {
 	validateEmptyReturn(optionName, chunk);
 
-	if (typeof chunk !== 'string' && !isBinary(chunk)) {
+	if (typeof chunk !== 'string' && !isUint8Array(chunk)) {
 		throw new TypeError(`The \`${optionName}\` option's function must yield a string or an Uint8Array, not ${typeof chunk}.`);
```

Comment:
`typeof chunk` will say `object` when Buffer is passed in. It may be a good idea to be a little bit more user-friendly and specifically detect Buffer and say that in the message.

Reply:
Fixed. :+1: